app/models/issue.rb

```diff
@@ -1,8 +1,10 @@
 class Issue < ApplicationRecord
+  belongs_to :user
+  attr_accessor :attachment
```

Comment:
this is needed? do you have field "attachement" in DB ? if not - you need to add it